packages/gatsby/src/query/queue.js

```diff
@@ -59,6 +59,16 @@ const createDevelopQueue = getRunner => {
   return new Queue(handler, queueOptions)
 }
 
+const createAppropriateQueue = (graphqlRunner, runnerOptions = {}) => {
+  if (process.env.NODE_ENV === `production`) {
+    return createBuildQueue(graphqlRunner, runnerOptions)
+  }
```

Comment:
This appears to be a semantic change. Before it wasn't checking the NODE_ENV flag and always calling the `createBuildQueue` and now it's conditionally calling `createDevelopQueue` instead. Was this one of the parts that was already committed but unused until now?

Reply:
In the old version, when running develop it explicitly created the develop queue in `startListeningToDevelopQueue`, which was then used in the notorious `API_RUNNING_QUEUE_EMPTY` listener. Now we're using the same queue for the first build and subsequent builds, so we want the develop on in production because it has nice stuff like prioritising active pages, and sending the updated data to the websocket.